ops/charm.py

```diff
@@ -1350,6 +1351,8 @@ def on(self) -> CharmEvents: ...  # noqa
     def __init__(self, framework: Framework):
         super().__init__(framework, None)
 
+        self._static_events: Set[str] = set(self.on.events())
```

Comment:
Off the top of my head I'm not sure what static means here.

- events like "start" -- sure
- events on relations that we pick up from metadata?
- custom events -- surely not

Perhaps a comment would help me.

Reply:
Added a comment.